src/views/HomeView.vue

```diff
@@ -1,5 +1,9 @@
-<script setup lang="ts"></script>
-
 <template>
-  <main></main>
+  <main>
+    <TitleItem title="Title" />
```

Comment:
By Vue convention it is preferred to use kebab-case when using components in the template: 
```<title-component />```